features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/tracing/ConfigureTracingView.kt

```diff
@@ -0,0 +1,250 @@
+import io.element.android.libraries.designsystem.theme.components.Text
+import io.element.android.libraries.designsystem.theme.components.TopAppBar
+import io.element.android.libraries.matrix.api.tracing.LogLevel
+import io.element.android.libraries.matrix.api.tracing.Target
+import io.element.android.libraries.theme.ElementTheme
+import kotlinx.collections.immutable.ImmutableMap
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ConfigureTracingView(
+    state: ConfigureTracingState,
+    onBackPressed: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var showMenu by remember { mutableStateOf(false) }
+    Scaffold(
+        modifier = modifier
+            .fillMaxSize()
+            .systemBarsPadding()
+            .imePadding(),
+        contentWindowInsets = WindowInsets.statusBars,
+        topBar = {
+            TopAppBar(
+                navigationIcon = {
+                    BackButton(onClick = onBackPressed)
+                },
+                title = {
+                    Text(
+                        text = "Configure tracing",
+                        style = ElementTheme.typography.aliasScreenTitle,
+                        maxLines = 1,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                },
+                actions = {
+                    IconButton(
+                        onClick = { showMenu = !showMenu }
+                    ) {
+                        Icon(
+                            imageVector = Icons.Default.MoreVert,
+                            tint = ElementTheme.materialColors.secondary,
+                            contentDescription = null,
+                        )
+                    }
+                    DropdownMenu(
+                        expanded = showMenu,
+                        onDismissRequest = { showMenu = false }
+                    ) {
+                        DropdownMenuItem(
+                            onClick = {
+                                showMenu = false
+                                state.eventSink.invoke(ConfigureTracingEvents.ResetFilters)
+                            },
+                            text = { Text("Reset to default") },
+                            leadingIcon = {
+                                Icon(
+                                    Icons.Outlined.Delete,
+                                    tint = ElementTheme.materialColors.secondary,
+                                    contentDescription = null,
+                                )
+                            }
+                        )
+                    }
+                }
+            )
+        },
+        content = {
+            Column(
+                modifier = Modifier
+                    .padding(it)
+                    .consumeWindowInsets(it)
+                    .verticalScroll(state = rememberScrollState())
+            ) {
+                CrateListContent(state)
+                ListItem(
```

Comment:
Could we add a button here to actually restart the app? I remember we had something similar in EA.

Reply:
I though about it, but I do not really see a way to do that properly. The Application `ElementXApplication` has to be restarted. Calling `System.exit(0)` may work, but will not restart the app. We could use the `AlarmManager` to schedule a restart, but as you said, not sure we want to bother.

The most annoying thing is that this screen is only available on debug build. Could be nice to have them on nightly. But this will not come with this PR.